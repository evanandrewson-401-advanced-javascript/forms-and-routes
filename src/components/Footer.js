import React from 'react';

const Footer = () => {
  return (
    <footer>
      <p>&copy; Evan inc.</p>
    </footer>
  );
};

export default Footer;
