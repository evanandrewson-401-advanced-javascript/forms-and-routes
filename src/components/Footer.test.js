import Footer from './Footer';
import React from 'react';
import { shallow } from 'enzyme';

describe('Footer component', () => {
  it('renders a footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
