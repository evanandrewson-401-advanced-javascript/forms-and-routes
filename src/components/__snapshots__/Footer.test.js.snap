// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders a footer 1`] = `
<footer>
  <p>
    Â© Evan inc.
  </p>
</footer>
`;
